package.json

```diff
@@ -53,31 +58,34 @@
     "aws-lambda": "^0.1.2",
     "aws-sdk": "^2.562.0",
     "aws-xray-sdk": "^2.4.0",
+    "liquibase": "^1.0.1",
     "moment": "^2.24.0",
+    "mysql2": "^2.2.5",
     "node-yaml": "^3.2.0",
     "reflect-metadata": "^0.1.13",
     "request": "^2.88.0",
     "request-promise": "^4.2.4",
+    "testcontainers": "^7.6.2",
     "ts-node-register": "^1.0.0"
   },
   "devDependencies": {
     "@types/aws-lambda": "^8.10.34",
-    "@types/jest": "^24.0.21",
+    "@types/jest": "^26.0.0",
     "@types/jest-plugin-context": "^2.9.2",
-    "@types/node": "^12.12.5",
+    "@types/node": "^12.20.7",
     "@types/request-promise": "^4.1.44",
     "@types/sinon": "^7.5.0",
     "audit-filter": "^0.5.0",
     "aws-lambda-mock-context": "^3.2.1",
     "gulp": "^4.0.2",
     "husky": "^3.0.9",
-    "jest": "^24.9.0",
+    "jest": "^26.0.0",
     "jest-plugin-context": "^2.9.0",
     "jest-sonar-reporter": "^2.0.0",
     "lambda-tester": "^3.5.0",
     "serverless": "^2.24.0",
     "serverless-dependency-invoke": "^0.0.9",
-    "serverless-offline": "^5.12.0",
+    "serverless-offline": "^6.9.0",
```

Comment:
I think it is best if we use the save-exact when saving packages, we are not sure whether all the packages we use follow the semver convention and not locking a specific version could break our app if breaking changes were pushed on patch, minor version for example.

We can also add dependabot for upgrading packages

Reply:
I'm not adding dependabot as we need to make the release cut, but feel free to change this on a future branch. However, I have set all versions to exact on [this commit](https://github.com/dvsa/cvs-tsk-update-store/pull/1/commits/62cf5de601f82c513403092b83197f755092bd5e)